core/src/main/scala/org/apache/spark/scheduler/SchedulerBackend.scala

```diff
@@ -49,4 +49,11 @@ private[spark] trait SchedulerBackend {
    */
   def applicationAttemptId(): Option[String] = None
 
+  /**
+   * Get the URLs for the driver logs. These URLs are used to display the links in the UI
+   * Executors tab for the driver.
+   * @return The urls to the logs of the driver
```

Comment:
nit: should be more descriptive. e.g. "Map containing the log names and their respective URLs."